ui/lib/pki/addon/components/pki-role-generate.hbs

```diff
@@ -60,7 +72,7 @@
           disabled={{this.save.isRunning}}
           data-test-pki-generate-button
         >
-          Generate
+          {{capitalize this.verb}}
```

Comment:
I'd have a tough time finding this template if I was looking for where certificates are signed. maybe something like `pki-role-certificate.js` 🤔, but probably fine to leave as is, just wanted to mention it!

wish the docs said "issue" certificate instead of "generate" then I'd feel good about changing the `generate.js` -> `issue.js` to match the API endpoints, and reserving the word generate for doing just that!

Reply:
Yeah, maybe instead this needs to be `pki-certificate-generate` since that's the outcome? But that leaves out the sign option entirely 🤔 _words are hard_

Reply:
They ARE. We could add `-sign` at the end....but that's a long file name. Or remove `pki-` and nest it in a `role/` folder? 

potato potato 🥔  - I think fine as as is 😅